rules/utils.js

```diff
@@ -8,7 +8,14 @@ function traverse(node, test) {
         break
       case 'MemberExpression':
         node = node.object
-        if (test(node.property)) return true
+        if (node.property) {
+          if (node.property.type === 'Identifier') {
```

Comment:
Identifier here, refers to `example` in `$div.example.foo()`, right? If so, a inline comment would help.

Same for the Literal below, which I think would be `[0]` but reaching complexity where I'm not sure from a static scan alone.

Reply:
Fixed